src/Search_Replace_Command.php

```diff
@@ -801,13 +808,16 @@ private function log_bits( $search_regex, $old_data, $old_matches, $new ) {
 			$diff += strlen( $new ) - strlen( $old_matches[0][ $i ][0] );
 			$i++;
 			return $new;
-		}, $old_data );
+		}, $old_data, $this->regex_limit );
 
 		$old_bits = $new_bits = array();
 		$append_next = false;
 		$last_old_offset = $last_new_offset = 0;
 		$match_cnt = count( $old_matches[0] );
```

Comment:
I think the easiest thing is just to do
```
$match_cnt = count( $new_matches[0] ); // `$new_matches[0]` may be limited by `regex_limit` so use instead of `$old_matches[0]`.
```
here which means the `empty()` checks following aren't needed.

Reply:
@gitlost why not use the `$count` `preg_replace_callback` provides?

Reply:
Er, well one could, but it's not important and I don't think reviewing a review is going to work to be honest! If you'd like to contribute please create an idea or an issue or a PR.

Reply:
On second thoughts you're right @alpipego that is a nicer way to do it so as suggested in https://github.com/wp-cli/search-replace-command/pull/70 @miya0001 you can just do `}, $old_data, $this->regex_limit, $match_cnt );` at line 811 instead.

Reply:
>  don't think reviewing a review is going to work

Of course you're right. The context is, that I am going through this with @miya0001 at my side at WordCamp Bangkok Contributer Day with the goal of fixing his PR.

Reply:
Ah I see! Sorry, I thought the PR was being hi-jacked! Ignore my previous comments!